src/main.rs

```diff
@@ -259,10 +299,36 @@ async fn main() {
 
     // creates genesis and writes to binary file
     match genesis.generate(solana_root.get_root_path(), &build_path) {
-        Ok(_) => (),
+        Ok(_) => info!("Created genesis successfully"),
         Err(err) => {
-            error!("generate genesis error! {}", err);
+            error!("generate genesis error! {err}");
             return;
         }
     }
+
+    //unwraps are safe here. since their requirement is enforced by argmatches
+    let docker = DockerConfig::new(
+        matches
+            .value_of("base_image")
+            .unwrap_or_default()
```

Comment:
nit: This should probably be an `unwrap()`, like all the others. Even though it won't get triggered thanks to `default_value(...)`